Detectors/HMPID/workflow/src/DigitReaderSpec.cxx_notused.cxx

```diff
@@ -19,6 +19,7 @@
 #include "Framework/ConfigParamRegistry.h"
 #include "DataFormatsParameters/GRPObject.h"
 #include "Framework/Logger.h"
+#include "FairLogger.h" // for LOG
```

Comment:
why `Framework/Logger.h` is not enough?

Reply:
in fact, if this is not used, please suppress it

Reply:
I removed the cxx file